cumulusci/core/config/OrgConfig.py

```diff
@@ -19,6 +22,23 @@ def __init__(self, config, name):
         super(OrgConfig, self).__init__(config)
 
     def refresh_oauth_token(self, keychain, connected_app=None):
+        SFDX_CLIENT_ID = os.environ.get("SFDX_CLIENT_ID")
+        SFDX_HUB_KEY = os.environ.get("SFDX_HUB_KEY")
+        if SFDX_CLIENT_ID and SFDX_HUB_KEY:
+            login_url = (
+                "https://test.salesforce.com"
```

Comment:
Would it make sense to have `SBX_LOGIN_URL` and `PROD_LOGIN_URL` attributes on `BaseConfig`?

Reply:
I guess we could turn them into constants somewhere, and that could be helpful if someone wants to use CumulusCI with one of our internal test environments or when running the core application locally. I'd rather work on that in a separate PR when we can get input from someone who actually needs to do that, though.